lib/browser/api/screen.ts

```diff
@@ -1,38 +1,41 @@
+import { EventEmitter } from 'events';
+
 const { createScreen } = process._linkedBinding('electron_common_screen');
 
 let _screen: Electron.Screen;
 
+const ensureScreen = () => {
+  _screen = createScreen();
+  EventEmitter.call(_screen);
```

Comment:
Is this `EventEmitter` constructor call necessary? We didn't need it before.

Reply:
@nornagon we did, it was done previously in `createLazyInstance`

Reply:
that was prior to the ginification though, yes? I don't believe the new code needs it.

Reply:
i'll check!